### ISSUE TITLE
Rimozione di soggetto da procedimento in corso

### ISSUE THREAD

**Issue by busaster** [italia/anpr#511]
Ambiente di Test.
Ho aperto un procedimento di cambio residenza per due persone (id ANPR 724040)
Ho poi perfezionato la mutazione di residenza (id ANPR 724053).
A quel punto ho rigettato una delle due mutazioni (id ANPR 724055) e ho rimosso la persona dal procedimento (id ANPR 724056).

Ho testato quindi il risultato producendo un certificato anagrafico di nascita per il soggetto rigettato (id ANPR 724058) ma contro le mie aspettative sono stato bloccato dall'errore "Il certificato non puo' essere emesso perche' risulta un procedimento aperto per il soggetto intestatario del certificato".

Come mai ?

**Comment by gcarbonin** [comment]
Il procedimento è stato chiuso?

**Comment by busaster** [comment]
No, il procedimento non è stato chiuso. Ma avendo rimosso il soggetto dal procedimento il fatto che questo sia ancora aperto non dovrebbe essere ininfluente ?

**Comment by gcarbonin** [comment]
Il servizio controlla se il procedimento è aperto o meno

**Comment by busaster** [comment]
Ho capito cosa fa oggi il controllo, ma mi non pare corretto che lavori così' per un soggetto rimosso dal procedimento.

**Comment by azabatta** [comment]
Buonasera, 
la modifica richiesta è stata apportata ed estesa in tutti gli ambienti.
Restiamo in attesa di un cortese riscontro.

**Comment by busaster** [comment]
Ho ripetuto il test iniziale, ora dopo il rigetto parziale sono riuscito a produrre un certificato.
Quindi problema risolto, chiudo la issue.
Grazie
